chain/consensus/hierarchical/subnet/manager/manager.go

```diff
@@ -350,12 +351,29 @@ func BuildSubnetMgr(mctx helpers.MetricsCtx, lc fx.Lifecycle, s *SubnetMgr) {
 	})
 }
 
+func (s *SubnetMgr) GetActorState(ctx context.Context, id address.SubnetID, actor address.Address) (*subnet.SubnetState, error) {
```

Comment:
As discussed sync, it is better to use `api := GetSubnetAPI(...)` and from there `GetActorState`, than exposing this function in all APIs (from the root to ever subnet) publically.